doc/examples/realtime_servo/realtime_servo_tutorial.rst

```diff
@@ -13,10 +13,12 @@ This tutorial shows how to use MoveIt Servo to send real-time servo commands to
 
 Getting Started
 ---------------
+
 If you haven't already done so, make sure you've completed the steps in :doc:`Getting Started </doc/tutorials/getting_started/getting_started>`.
 
 Launching a Servo Node
 ----------------------
+
```

Comment:
Also, what is with these new-lines? Do you have some feature of your editor auto-formatting this?

If this feature is generally useful you should add it to pre-commit with a separate change.

Reply:
Those were manual changes.